render/jpql/src/test/kotlin/com/linecorp/kotlinjdsl/render/jpql/serializer/impl/JpqlJoinedEntitySerializerTest.kt

```diff
@@ -0,0 +1,51 @@
+package com.linecorp.kotlinjdsl.render.jpql.serializer.impl
+
+import com.linecorp.kotlinjdsl.querymodel.jpql.from.impl.JpqlJoinedEntity
+import com.linecorp.kotlinjdsl.render.TestRenderContext
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlRenderSerializer
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlSerializerTest
+import com.linecorp.kotlinjdsl.render.jpql.writer.JpqlWriter
+import io.mockk.*
+import io.mockk.impl.annotations.MockK
+import org.assertj.core.api.WithAssertions
+import org.junit.jupiter.api.Test
+
+@JpqlSerializerTest
+class JpqlJoinedEntitySerializerTest : WithAssertions {
+    private val sut = JpqlJoinedEntitySerializer()
+
+    @MockK
+    private lateinit var writer: JpqlWriter
+
+    @MockK
+    private lateinit var serializer: JpqlRenderSerializer
+
+    @Test
+    fun handledType() {
+        // when
+        val actual = sut.handledType()
+
+        // then
+        assertThat(actual).isEqualTo(JpqlJoinedEntity::class)
+    }
+
+    @Test
+    fun `serialize - WHEN joined entity is given, THEN draw full syntax`() {
+        // given
+        every { writer.write(any<String>()) } just runs
+        every { serializer.serialize(any(), any(), any()) } just runs
+
+        val part = mockkClass(type = JpqlJoinedEntity::class, relaxed = true)
```

Comment:
I've noticed that the class `JpqlJoinedEntity` isn't marked an internal class, 
unlike the other `impl` classes which are internal. 
If it's intended for `JpqlJoinedEntity` to also be internal, I can make that adjustment in this PR :)

Reply:
As I've been experimenting with different implementations, there are some classes that I've accidentally left out the internal annotation..

The classes in the impl package in the Query-Model module are all internal classes. So I would be happy if you could add the internal annotation to `JpqlJoinedEntity` as well.